app/controllers/api/internal/v1/laa_references_controller.rb

```diff
@@ -9,6 +9,8 @@ def create
           if contract.errors.present?
             render json: contract.errors.to_hash, status: :bad_request
           else
+            make_common_platform_calls(contract)
```

Comment:
I believe this can be extracted out to a service that we can then call as a job:
```
LaaReferenceUpdaterJob.perform(contract)
```